web_image_viewer/README.rst

```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============================
+View image in odoo without download to local store
+=============================
+
+This module was written to view image directly in odoo web client 
+without download to local store
+
+Usage
+=====
+    Install addon [Settings > Modules > Install Modules]
```

Comment:
These are not usage instructions. Specify where you can find an example for seeing an image.